tests/cumulative_links.csv

```diff
@@ -0,0 +1,7 @@
+strata,reference,target,period,forward_imputation_link,backward_imputation_link,imputation_group,cumulative_forward_imputation_link,cumulative_backward_imputation_link
+100,100000,200,202402,1,2,1,1,2
+100,100000,,202403,2,0.6,2,2,0.6
+100,100000,,202404,3,1,2,6,1
+200,100001,,202402,1,4,3,1,2
+200,100001,,202403,3,0.5,3,3,0.5
+200,100001,300,202404,0.5,1,4,0.5,1
```

Comment:
I might be misunderstanding the code / method, but why isn't cumulative_forward_imputation_link for the last row 1.5. Is that because it is in a different imputation group?

Reply:
yes that is correct. We only want to multiply the links where there are consecutive missing values. In fact, maybe we should turn all cumulative links in the same row as a return to nan because it has no meaning.